### PR TITLE
Add hippynn

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jan-janssen** [comment]
The windows tests fail, as `PyTorch` is not available for windows, still I would like this package to be `noarch`: 
```
The reported errors are:
- Encountered problems while solving:
-   - nothing provides requested pytorch
- 
```

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

**Comment by dopplershift** [comment]
I can't make sense of this license:
```


Copyright 2019. Triad National Security, LLC. All rights reserved.
This program was produced under U.S. Government contract 89233218CNA000001 for Los Alamos
National Laboratory (LANL), which is operated by Triad National Security, LLC for the U.S.
Department of Energy/National Nuclear Security Administration. All rights in the program are
reserved by Triad National Security, LLC, and the U.S. Department of Energy/National Nuclear
Security Administration. The Government is granted for itself and others acting on its behalf a
nonexclusive, paid-up, irrevocable worldwide license in this material to reproduce, prepare
derivative works, distribute copies to the public, perform publicly and display publicly, and to permit
others to do so.

This program is open source under the BSD-3 License.
Redistribution and use in source and binary forms, with or without modification, are permitted
provided that the following conditions are met:
1. Redistributions of source code must retain the above copyright notice, this list of conditions and
the following disclaimer.

2.Redistributions in binary form must reproduce the above copyright notice, this list of conditions
and the following disclaimer in the documentation and/or other materials provided with the
distribution.

3.Neither the name of the copyright holder nor the names of its contributors may be used to endorse
or promote products derived from this software without specific prior written permission.

THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS
IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR
CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR
OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```

Are the rights reserved "by Triad National Security, LLC, and the U.S. Department of Energy/National Nuclear
Security Administration. The Government is granted for itself and others acting on its behalf a
nonexclusive, paid-up, irrevocable worldwide license" or is it BSD3? IANAL, but those two would seem to be in conflict.

**Comment by jan-janssen** [comment]
> Are the rights reserved "by Triad National Security, LLC, and the U.S. Department of Energy/National Nuclear Security Administration. The Government is granted for itself and others acting on its behalf a nonexclusive, paid-up, irrevocable worldwide license" or is it BSD3? 

It is basically a dual license, BSD3 for everybody and special rights for the government. We are required to add this part to the license files when we release code as open source.

**Comment by jan-janssen** [comment]
@dopplershift and @bastianZim - Can one of you take another look at this pull request?

**Comment by BastianZim** [comment]
I'm happy with the license, but I think it should be a custom one (LicenseRef-Something): https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx-identifiers-and-expressions

Also, this is a beta build, so it should target a pre-release: https://conda-forge.org/docs/maintainer/knowledge_base.html#creating-a-pre-release-build

**Comment by jan-janssen** [comment]
> I'm happy with the license, but I think it should be a custom one (LicenseRef-Something): https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx-identifiers-and-expressions

I am a bit confused, it is a dual license. Is there a specific description for dual licenses? 

> Also, this is a beta build, so it should target a pre-release: https://conda-forge.org/docs/maintainer/knowledge_base.html#creating-a-pre-release-build

We are just about to release the first official version, just waiting to make sure everything is fine with conda-forge.

**Comment by BastianZim** [comment]
> it is a dual license

Ahh, I wasn't sure. Then something like this should work: `LicenseRef-US-Government OR BSD-3-Clause`.

> We are just about to release the first official version, just waiting to make sure everything is fine with conda-forge.

Ok, should I wait with merging then until the official version is released? Otherwise, it would still need to target the beta channel but you can just remove that in the feedstock when you switch to the normal release.

**Comment by jan-janssen** [comment]
> > it is a dual license
> 
> Ahh, I wasn't sure. Then something like this should work: `LicenseRef-US-Government OR BSD-3-Clause`.
> 
> > We are just about to release the first official version, just waiting to make sure everything is fine with conda-forge.
> 
> Ok, should I wait with merging then until the official version is released? Otherwise, it would still need to target the beta channel but you can just remove that in the feedstock when you switch to the normal release.

Ok, I updated both - it would be great if we can merge it quickly, that helps me to convince the internal stakeholders, to push our public software also to conda-forge.

File: recipes/hippynn/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+channel_targets:
+  - conda-forge hippynn_dev
```
Comment (BastianZim) [inline recipes/hippynn/conda_build_config.yaml:2]:
```suggestion
  - conda-forge hippynn_rc
```
See here: https://conda-forge.org/docs/maintainer/knowledge_base.html#pre-release-builds

**Comment by BastianZim** [comment]
Just waiting for CI to pass. :)

**Comment by BastianZim** [comment]
FYI though https://github.com/conda-forge/status/issues/135

**Comment by jan-janssen** [comment]
> Just waiting for CI to pass. :)

It is ready ;-)

**Pull request merged**